dashboard/src/utils/utils.ts

```diff
@@ -13,6 +13,8 @@ import type {
 import { sanitizeTableValue } from '@/components/Table/tableUtils';
 import type { ISummaryItem } from '@/components/Tabs/Summary';
 
+const UNKNOWN_STRING = 'Unknown';
```

Comment:
to make it a little clear that this is from backend move to `constants/backend.ts` 
not the best solution but a little improvement